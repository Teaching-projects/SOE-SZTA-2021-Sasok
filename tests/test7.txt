10
0
1.1
14
10
3.4