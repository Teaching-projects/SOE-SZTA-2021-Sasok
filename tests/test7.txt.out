Első egység HP-ja: Első egység DMG-je: Második egység HP-ja: Második egység DMG-je: 
A Csata elkezdődött! Warrior kezdheti a támadást Shaman ellen.
1. körben Warrior támad --- Warrior élete: 15 || Shaman élete: 14
2. körben Shaman támad, ---Warrior élete: 5 || Shaman élete: 14
2. körben Warrior támad --- Warrior élete: 5 || Shaman élete: 14
3. körben Shaman támad, ---Warrior élete: -5 || Shaman élete: 14

SHAMAN GYOZEDELMESKEDETT!
